### PR TITLE
[KOGITO-7324] - Add the CONTRIBUTIONS.md

### PR DESCRIPTION
**Pull request by ricardozanini**
Signed-off-by: Ricardo Zanini <zanini@redhat.com>

In this PR, we introduce the CONTRIBUTIONS doc.

**JIRA:** https://issues.redhat.com/browse/KOGITO-7324

Please make sure that your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [ ] The nav.adoc file has a link to this guide in the proper category

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
+# Contributing
+
+We accept all kinds of contributions:
+
+1. Reviewing a PR
+2. Opening [an issue](issues) by describing what problem you see that we need to fix
```
Comment (spolti) [inline CONTRIBUTING.md:6]: are we going to use github tot rack issues?

**Review by spolti — APPROVED** [review]

**Review by hbelmiro — COMMENTED** [review]
Nice job, @ricardozanini!
I left two comments about typos.

**Review by hbelmiro — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+```
+
+More details at [Antora documentation](https://docs.antora.org/antora/latest/playbook/asciidoc-attributes)
+
+### Using Dry URLs for Links
+
+You should assign the URL to a short, easy to remember attribute. For example:
+
+```asciidoc
+issues_url: https://github.com/asciidoctor/asciidoctor/issues`
```
Comment (hmanwani-rh) [inline CONTRIBUTING.md:134]:
```suggestion
:issues_url: https://github.com/asciidoctor/asciidoctor/issues`
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+More details at [Antora documentation](https://docs.antora.org/antora/latest/page/include-a-page).
+
+### Assigning attributes on a site
+
+You can create attributes in `{project_root}/{component_name}/antora.yml` file. These attributes can be use anywhere in that module.
+
+```yaml
+asciidoc:
+  attributes:
+    example_url: https://www.myexample.com
```
Comment (hmanwani-rh) [inline CONTRIBUTING.md:124]:
```suggestion
    :example_url: https://www.myexample.com
```

**Review by hmanwani-rh — COMMENTED** [review]
@ricardozanini Thanks for creating this contributing page.
Added a few suggestions, please take a look and let me know what you think

File: README.adoc [hunk 4]
```diff
@@ -1,4 +1,4 @@
-= Kogito Documentation and Guides
+= Kogito Guides
```
[the remark this answers is not in the document]
Comment (ricardozanini) [reply README.adoc:1]: This is a more generic contributions doc that we can use for any other guides within this repo. That's why I kept it generic.

**Review by hmanwani-rh — APPROVED** [review]

**Pull request merged**